php_phongo.c

```diff
@@ -483,41 +482,6 @@ PHP_FUNCTION(MongoDB_disabled___wakeup) /* {{{ */
 /* }}} */
 
 /* {{{ Extension functions, module dependencies, and module entry */
-ZEND_BEGIN_ARG_INFO_EX(ai_bson_fromPHP, 0, 0, 1)
```

Comment:
Should you remove "Extension functions" from the comment above?

Reply:
Good point, done.